tools/tcat_ble_client/cli/cli.py

```diff
@@ -57,12 +57,14 @@ def __init__(self,
             'network_name': GetNetworkNameCommand(),
             'ping': PingCommand(),
             'dataset': DatasetCommand(),
+            'get_dataset': ExtractDatasetCommand(),
             'thread': ThreadStateCommand(),
             'scan': ScanCommand(),
             'random_challenge': GetRandomNumberChallenge(),
             'present_hash': PresentHash(),
             'peer_pskd_hash': GetPskdHash(),
-            'tlv': TlvCommand()
+            'tlv': TlvCommand(),
+            'get_commisioner_cert': GetCommissionerCertificate(),
```

Comment:
Sorry for the new issue raised; seeing this just now. Propose to either use the correct spelling:

```suggestion
            'get_commissioner_cert': GetCommissionerCertificate(),
```

or to make it shorter which seems much better for CLI:

```suggestion
            'get_comm_cert': GetCommissionerCertificate(),
```

In the past I've used "comm cert" or "comm_cert" before as a shorthand and so far have been happy with that.

Reply:
I have changed the name to `get_comm_cert`